### ISSUE TITLE
Accessing SolveTime does not work

### ISSUE THREAD

**Issue by ohmsweetohm1** [JuliaOpt/Cbc.jl#138]
Retrieving the `SolveTime` works for e.g. `Gurobi` or `GLPK` like:

```julia
using Gurobi
model = JuMP.Model(Gurobi.Optimizer)
@variable(model, x >= 0)
@objective(model, Min, x)
optimize!(model)
JuMP.solve_time(model)
```

```julia
using GLPK
model = JuMP.Model(GLPK.Optimizer)
@variable(model, x >= 0)
@objective(model, Min, x)
optimize!(model)
JuMP.solve_time(model)
```
however, not for `Cbc`:

```julia
using Cbc
model = JuMP.Model(Cbc.Optimizer)
@variable(model, x >= 0)
@objective(model, Min, x)
optimize!(model)
JuMP.solve_time(model)
```

**Comment by blegat** [comment]
You should now be available in the v0.7 just release 11 hours ago.
See https://github.com/JuliaOpt/Cbc.jl/issues/124

**Comment by odow** [comment]
Can confirm this works on latest release.
